tensorboard/components/tf_tensorboard/test/tensorboardTests.html

```diff
@@ -19,7 +19,7 @@
 <html>
 <head>
   <link rel="import" href="../../polymer/polymer.html">
-  <script src="../../webcomponentsjs/webcomponents-lite.js"></script>
+  <script jscomp-nocmpile src="../../webcomponentsjs/webcomponents-lite.js"></script>
```

Comment:
typo: `nocmpile` → `nocompile`; did automated tests not catch this?

Reply:
Nope :(. Fixed it manually for now.